### PR TITLE
Add taco-0.6.0 tool

### PR DESCRIPTION
**Pull request by upendrak**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by upendrak** [comment]
Can you please remove recipes/picard/2.7.1/picard.sh from this PR.
Thanks

File: recipes/taco/0.6.0/build.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -eu
+
+mkdir -p $PREFIX/bin
+chmod a+x $SRC_DIR/taco
```
Comment (bgruening) [inline recipes/taco/0.6.0/build.sh]: You can just write `chmod a+x taco` and `cp taco $PREFIX/bin/`

Comment (upendrak) [inline recipes/taco/0.6.0/build.sh]: Ok got it..

**Comment by upendrak** [comment]
Ok shall i make another PR then?

**Comment by bgruening** [comment]
I guess so, or you can rebase against master after you have remove the recipe.

**Comment by upendrak** [comment]
Ok shall i close this PR then?

**Comment by upendrak** [comment]
Do you know why the build failed?

**Comment by bgruening** [comment]
This is the error: https://travis-ci.org/bioconda/bioconda-recipes/jobs/186235985#L539
Your are copying recompiled binaries that are not relocatable, so compiling it from source would fix it.

**Comment by bgruening** [comment]
@upendrak how can we help you here?

**Comment by bgruening** [comment]
I guess this can be closed, you try version 0.7 as I have seen.

**Pull request closed without merging**